content/docs/command-reference/import-url.md

```diff
@@ -37,6 +39,12 @@ external data source changes. Example scenarios:
 having to manually copy files from the supported locations (listed below), which
 may require installing a different tool for each type.
 
+When you don't actually want to store the whole data file / directory in your
+local workspace but rather import it directly to the remote storage,
+`--to-remote` option can be given. See the
+["import straight to remote"](#example-import-straight-to-the-remote) example
+for more details.
```

Comment:
Copy edit:
```suggestion
When you don't want to store the target data in your local system, you can still
create an import `.dvc` file while transferring a file or directory directly to
remote storage, by using the `--to-remote` option. See the
[Import straight to remote](#example-import-straight-to-remote) example for
more details.
```

Reply:
But in `add` we created a whole section, while in this ref. it's one new paragraph in the description + an example. Which option is better (simple=better)? Should we be consistent in both refs?

Reply:
After reviewing the example in `import-url` I think the change we did in that ref. makes more sense: just a paragraph in the cmd Description, and link to an example that explains more visually.

Reply:
So should I add the same example to the `add` and remove the section @jorgeorpinel?

Reply:
Yes, I now like the example option better, so there is space to explain the caveats there (and just add a summary paragraph about it in the description). Either way it could work as long as the changes in add and import-url are consistent. And it's OK if the examples are very similar, yes. Thanks

Reply:
Hi! I left a few last comments in the import-url Example so once that's done we can copy (adapt) it to add as well (: